src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -355,11 +356,17 @@ g.test('sampleCount,valid_sampleCount_with_other_parameter_varies')
     const info = kTextureFormatInfo[format];
     t.skipIfTextureFormatNotSupported(format);
     t.selectDeviceOrSkipTestCase(info.feature);
+    t.selectDeviceForRenderableColorFormatOrSkipTestCase(format);
   })
   .fn(t => {
     const { dimension, sampleCount, format, mipLevelCount, arrayLayerCount, usage } = t.params;
     const { blockWidth, blockHeight } = kTextureFormatInfo[format];
 
+    t.skipIf(
+      sampleCount === 4 && t.isCompatibility && (format === 'rgba16float' || is32Float(format)),
```

Comment:
Should this be `sampleCount > 1`? because we plan to add more sample counts?
Also, wdyt about adding `t.skipIfCanNotMultisampleFormat` as I'd expect it to come up in other places.